common/scala/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerFactory.scala

```diff
@@ -51,7 +51,9 @@ case class ContainerPoolConfig(userMemory: ByteSize, concurrentPeekFactor: Doubl
 
 case class RuntimesRegistryCredentials(user: String, password: String)
 
-case class RuntimesRegistryConfig(url: String, credentials: Option[RuntimesRegistryCredentials])
+case class RuntimesRegistryConfig(url: String,
+                                  includeUserImages: Option[Boolean] = None,
```

Comment:
Instea dof using an optional boolean it would be better to not define any value and let the default value (`false`) be defined in application.conf itself.

Reply:
note that this would be removed with a change above and you cons an instance of this class with a different url value.